hypervisor/arch/x86/assign.c

```diff
@@ -45,8 +45,8 @@ is_entry_active(struct ptdev_remapping_info *entry)
 
 static bool ptdev_hv_owned_intx(struct vm *vm, union source_id *virt_sid)
 {
-	/* vm0 pin 4 (uart) is owned by hypervisor under debug version */
-	if (is_vm0(vm) && (virt_sid->intx_id.pin == 4U)) {
+	/* vm0 pin 6 (uart) is owned by hypervisor under debug version */
+	if (is_vm0(vm) && (virt_sid->intx_id.pin == 6U)) {
```

Comment:
should we use COM1_IRQ here rather than 6U? 

BTW, how you let SOS know the vUART is in IRQ 6?

Reply:
Ok, let me use COM1_IRQ. 
SOS will do autoirq detection for UART